### PR TITLE
Fixes #229: add support for multi-sensor hanwa camera

### LINKED ISSUE #229: Hanwha multi-sensor
Example that shows sensor 1:

rtsp://username:password@10.0.0.153:554/1/profile2/media.smp

```
rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{channel}}/profile{{stream}}/media.smp
```

`channels` are `0;1;2;3`
(sensors 0-3)

`streams` are only `2` for now (as H.264)

OUI is `00:09:18`

See doc at:
https://support.hanwhasecurity.com/hc/en-us/articles/236372028-Camera-RTSP-URL

please fix this along with issue #196 to make `2` the default.

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

**Review by jcmaslan — APPROVED** [review]
thanks!

File: cameras/paths.csv [hunk 1]
```diff
@@ -70,3 +70,4 @@ Q-See,camera,QCW2MPSL,(38:AF:29),rtsp://{{username}}:{{password}}@{{ip_address}}
 OVISS,nvr,ProS,(FC:D2:B6|E4:F1:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/unicast/c{{channel}}-n/s{{stream}}/live,H.264,8554,0;1,1;2;3;4;5;6;7;8,main stream;sub stream,Channel {{channel}},1,0,admin,admin,,
 OVISS,camera,Camera,(FC:D2:B6|E4:F1:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/video{{stream}},H.264,8554,1;2;3,,Main;Sub;Third,,Sub,Main,admin,123456,,
 Salient,nvr,Complete View,(BE:FE:11),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/CompleteView/live?cameraid={{stream}},H.264,554,1;2;3;4;5;6;7;8,,Stream {{stream}},,2,1,admin,admin,,https://www.salientsys.com/files/support/Sony%20Camera%20Supplement.pdf
+Hanwa,camera,multi-sensor,(00:09:18),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{channel}}/profile{{stream}}/media.smp,H.264,554,2,0;1;2;3,Stream {{stream}},Channel {{channel}},2,2,admin,admin,,https://support.hanwhasecurity.com/hc/en-us/articles/236372028-Camera-RTSP-URL
```
Comment (jcmaslan) [inline cameras/paths.csv]: @jhallard Hanwha spelling

**Review by jcmaslan — CHANGES_REQUESTED** [review]

**Review by jcmaslan — APPROVED** [review]

**Comment by jcmaslan** [comment]
@jhallard I think the default username and password are:

admin
4321

**Comment by jhallard** [comment]
changed default pwd

**Comment by jcmaslan** [comment]
did you confirm a different value @jhallard ?
The user manual said 4321

**Review by jcmaslan — CHANGES_REQUESTED** [review]
4321

**Comment by jhallard** [comment]
I thought I had read a different value when looking through other sources but, you're right, it should be. 4321

**Comment by jhallard** [comment]
@jcmaslan I'll deploy this as soon as you approve

**Review by jcmaslan — APPROVED** [review]

**Pull request merged**